### PR TITLE
Database design

### PR DESCRIPTION
**Pull request by luisferyimi19**

### CODE REVIEW

File: db/migrate/20190925033515_create_quotation_service_and_references.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class CreateQuotationServiceAndReferences < ActiveRecord::Migration[6.0]
+  def change
+    create_table :quotation_services do |t|
+      t.decimal :amount
+      t.decimal :holgura
```
Comment (Chofo2003) [inline db/migrate/20190925033515_create_quotation_service_and_references.rb]: Holgura should be slack/play/space.

File: app/models/user.rb [hunk 2]
```diff
@@ -6,4 +6,6 @@ class User < ApplicationRecord
 
   belongs_to :role
   has_many :quotations
+  has_many :attachments
+  has_many :comments, as: :commentable
```
Comment (Chofo2003) [inline app/models/user.rb]: Not sure if commentable is required here.

File: db/migrate/20190925033515_create_quotation_service_and_references.rb (hunk 1 above)
Comment (luisferyimi19) [reply db/migrate/20190925033515_create_quotation_service_and_references.rb]: Changed to percent

File: app/models/user.rb (hunk 2 above)
Comment (luisferyimi19) [reply app/models/user.rb]: Changed to has_many :comments

**Pull request merged**